test.js

```diff
@@ -22,3 +23,29 @@ test('catch', async t => {
 
 	t.true(inRange(promise.time, 200, 250));
 });
+
+test.cb.serial('log', t => {
+	function myDelay(ms) {
+		return new Promise(resolve => {
+			setTimeout(resolve, ms);
+		});
+	}
+
+	const unhook = hookStd.stdout(output => {
+		unhook();
```

Comment:
How would you unsubscribe when it returns a promise? Not necessary when using the `once` option. Maybe with a method? `promise.unsubscribe()`.